### PR TITLE
Adding Telemetry info for Analytics-Swift and Analytics-Kotlin SDKs

### PR DESCRIPTION
**Pull request by MichaelGHSeg**
### Proposed changes

Adding info to Analytics-Swift SDK documentation about Telemetry and how to disable it.

### Merge timing
- When [Telemetry for Swift](https://github.com/segmentio/analytics-swift/pull/369) is released

### CODE REVIEW

File: src/connections/sources/catalog/libraries/mobile/apple/index.md [hunk 1]
```diff
@@ -165,6 +165,13 @@ If you proxy your events through the `apiHost` config option, you must forward t
 > warning ""
 > If you are using the Analytics iOS (Classic) SDK, you can find [the documentation here](/docs/connections/sources/catalog/libraries/mobile/ios). Many of the features available in the Analytics-Swift SDK are not available in the Analytics iOS (Classic) SDK.
 
+## Telemetry
+The Analytics-Swift SDK collects telemetry data on configuration and usage by default. This includes basic information on SDK setup, plugins and event types used, and basic error details. No personally identifiable information (PII) or event data is collected, and Segment downsamples the data to minimize traffic.
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/mobile/apple/index.md:169]:
```suggestion
The Analytics-Swift SDK collects telemetry data on configuration and usage by default. This includes basic information on SDK setup, plugins and event types used, and basic error details. Segment downsamples the data to minimize traffic and doesn't collect any personally identifiable information (PII) or event data.
```

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/index.md [hunk 2]
```diff
@@ -201,5 +201,12 @@ Analytics-Kotlin is built with extensibility in mind. Use the tools list below t
 > warning ""
 > If you are using the Analytics Android (Classic) SDK, you can find [the documentation here](/docs/connections/sources/catalog/libraries/mobile/android). Many of the features available in the Analytics Kotlin SDK are not available in the Analytics Android (Classic) SDK.
 
+## Telemetry
+Telemetry data related to the configuration and usage of the Analytics-Kotlin SDK is collected and sent to Segment by default.  This includes some basic configuration information, plugins and event types used, and basic error information.  No PII or event data is collected and the information is downsampled to reduce traffic.
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/mobile/kotlin-android/index.md:205]:
```suggestion
Telemetry data related to the configuration and usage of the Analytics-Kotlin SDK is collected and sent to Segment by default.  This includes some basic configuration information, plugins and event types used, and basic error information.  Segment downsamples the data to minimize traffic and doesn't collect any personally identifiable information (PII) or event data.
```

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/index.md [hunk 3]
```diff
@@ -201,5 +201,12 @@ Analytics-Kotlin is built with extensibility in mind. Use the tools list below t
...
+
+The Telemetry system can be disabled at any time by setting `Telemetry.enable = false`.
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/mobile/kotlin-android/index.md:207]:
```suggestion
You can disable telemetry at any time by setting `Telemetry.shared.enable = false`.
```

File: src/connections/sources/catalog/libraries/mobile/kotlin-android/index.md [hunk 4]
```diff
@@ -201,5 +201,12 @@ Analytics-Kotlin is built with extensibility in mind. Use the tools list below t
...
+
+When internal errors or errors from plugins occur, the writekey may be included with the error data to help customers track down problems.  This can be disabled by setting `Telemetry.sendWriteKeyOnError = false`.
```
Comment (stayseesong) [inline src/connections/sources/catalog/libraries/mobile/kotlin-android/index.md:209]:
```suggestion
When internal errors or errors from plugins occur, the writekey may be included with the error data to help customers track down problems.  You can disable this by setting `Telemetry.sendWriteKeyOnError = false`.
```

**Review by stayseesong — APPROVED** [review]

**Comment by stayseesong** [comment]
@MichaelGHSeg when will Telemetry for Swift be live?

**Comment by MichaelGHSeg** [comment]
@stayseesong As soon as I go hit about twenty buttons, I was holding off for this process.  We can go ahead and release this now.  Thanks!

**Comment by stayseesong** [comment]
@MichaelGHSeg this will go live once our deploy goes out later today!

**Pull request merged**